### DISCUSSION TITLE
Rustdesk 1.2.0 and later crashes with error 0xc0000409

### DISCUSSION THREAD

**Discussion by OddUnicorn** [rustdesk/rustdesk#5495] [General]
### Bug Description

Trying to install rustdesk 1.2.2 on a remote PC currently running rustdesk 1.1.9, but it crashes before displaying anything.
Eventmanager shows this error:

<Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
<System>
  <Provider Name="Application Error" /> 
  <EventID Qualifiers="0">1000</EventID> 
  <Version>0</Version> 
  <Level>2</Level> 
  <Task>100</Task> 
  <Opcode>0</Opcode> 
  <Keywords>0x80000000000000</Keywords> 
  <TimeCreated SystemTime="2023-08-23T08:33:08.4798680Z" /> 
  <EventRecordID>51396</EventRecordID> 
  <Correlation /> 
  <Execution ProcessID="0" ThreadID="0" /> 
  <Channel>Application</Channel> 
  <Computer>PC1</Computer> 
  <Security /> 
  </System>
 <EventData>
  <Data>rustdesk.exe</Data> 
  <Data>1.2.2.0</Data> 
  <Data>64e417cd</Data> 
  <Data>librustdesk.dll</Data> 
  <Data>0.0.0.0</Data> 
  <Data>64e41708</Data> 
  <Data>c0000409</Data> 
  <Data>0000000000820d36</Data> 
  <Data>4090</Data> 
  <Data>01d9d59c7125bc90</Data> 
  <Data>C:\Users\User\AppData\Local\rustdesk\rustdesk.exe</Data> 
  <Data>C:\Users\User\AppData\Local\rustdesk\librustdesk.dll</Data> 
  <Data>a1c23784-7a60-4641-bdb0-8a59450e34f7</Data> 
  <Data /> 
  <Data /> 
  </EventData>
  </Event>
  
I tried to fix potential errors with all available options in "Reset Windows Update Tool", but the tool couldn't find anything that requires some sort of repair. Reverting back to rustdesk 1.1.9 for now, all later version are showing above behaviour. Rustdesk 1.2.0 and later is the only program that is having issues on this particular PC.

### How to Reproduce

Starting rustdesk 1.2.0 or later.

### Expected Behavior

Rustdesk should start without crashing.

### Operating system(s) on local side and remote side

Windows 10 -> Windows 10

### RustDesk Version(s) on local side and remote side

1.2.2 -> 1.1.9

### Screenshots

No screenshots available.

**Comment by rustdesk** [comment]
https://www.google.com/search?q=0xc0000409&sourceid=chrome-mobile&ie=UTF-8

**Comment by twooclock** [comment]
Had this same problem. There was an instance of RustDesk.exe running, after killing it in Task Manager everything started to work fine...!
